docs/api/README.md

```diff
@@ -17,7 +17,7 @@
 
 ## Sample Style Object
 
-```as
```

Comment:
The `as` thing is a hack for improved JSX support (otherwise syntax highlighting gets broken whenever it encounters any JSX in `js` mode) (unless they fixed that, in which case never mind!)